apps/backend/src/users/users.service.ts

```diff
@@ -1,17 +1,55 @@
-import { Injectable } from '@nestjs/common';
+import {
+  BadRequestException,
+  UnauthorizedException,
+  Injectable,
+} from '@nestjs/common';
 import { InjectRepository } from '@nestjs/typeorm';
-import { Repository } from 'typeorm';
+import { MongoRepository } from 'typeorm';
 
 import { User } from './user.entity';
+import { Status } from './types';
+import { ObjectId } from 'mongodb';
 
 @Injectable()
 export class UsersService {
   constructor(
     @InjectRepository(User)
-    private usersRepository: Repository<User>,
+    private usersRepository: MongoRepository<User>,
   ) {}
 
-  findAll(): Promise<User[]> {
-    return this.usersRepository.find();
+  async findAll(getAllMembers: boolean): Promise<User[]> {
+    if (!getAllMembers) return [];
+
+    const exampleUser: User = {
+      userId: new ObjectId('a0f3efa0f3efa0f3efa0f3ef'),
+      status: Status.ADMIN,
+      firstName: 'jimmy',
+      lastName: 'jimmy2',
+      email: 'jimmy.jimmy2@mail.com',
+      profilePicture: null,
+      linkedin: null,
+      github: null,
+      team: null,
+      role: null,
+    };
+
+    if (exampleUser.status == Status.APPLICANT) {
```

Comment:
:pick: Can we use `===` here and on line 48? Generally, there's no reason to prefer `==` over `===` (with one common exception of `variable == null` as a shortcut for `variable === null || variable === undefined`) and will only lead to not-so-obvious bugs:
```js
console.log([1] == true) // true
console.log([2] == true) // false
console.log(![] == []) // true
```